### ISSUE TITLE
How to change zsh-autocomplete color or make text item list under like zsh-autosuggestions

### ISSUE THREAD

**Issue by scila1996** [ohmyzsh/ohmyzsh#9728]
Hi. Current when i press '[TAB]' and it will be show the list under command like this. But color of them is alternative to command 

![image](https://user-images.githubusercontent.com/17109300/110980806-d531a400-8398-11eb-940e-f03b4f3b0ddf.png)

I like all item of under list will colored like zsh-autosuggestions

![image](https://user-images.githubusercontent.com/17109300/110980934-fb574400-8398-11eb-8fd1-47751abaf194.png)

How can i do that , thank all !

**Comment by mcornella** [comment]
The setting to change completion colors is `list-colors`, set using `zstyle`: https://stackoverflow.com/a/23568183/5798232.

Since `list-colors` uses ANSI color codes we need to get the ANSI color code for what `zsh-autosuggestions` uses. The default setting is `fg=8`, which is equivalent to saying `%F{8}` in prompt expansion sequence. Running `print -P %F{8}colored-text%f` gets us the same color, and looking at the hexdump of the output we see that the ANSI color sequence is `\e[90m`.

So to recap, the ANSI color code here is `90`. So the setting we want is
```
zstyle ':completion:*'  list-colors '=*=90'
```
Note that this will change the colors of suggested files and directories, in case you have that enabled. If you want to only apply that to certain command completions, look at the linked Stack Overflow answer for more options.
